### ISSUE TITLE
autocomplete: Chromevox reads 'Status' as you type or when focus is changed in the dropdown

### ISSUE THREAD

**Issue by m1h1r0** [angular/material#10896]
**Actual Behavior**:
 -  `What is the issue? *` 
Chromevox reads 'Status' as you type in a md-autocomplete when verbose description is enabled
 -  `What is the expected behavior?` 
Chromevox should read the messages about the matches

**CodePen** (or steps to reproduce the issue): *
Steps: 
- start ChromeVox, enable verbose description
- go to https://material.angularjs.org/latest/demo/autocomplete, type in the Basic Usage Demo

**AngularJS Versions**: *
 -  `AngularJS Version:` 1.6
 -  `AngularJS Material Version:` 1.1.0

**Additional Information**:
 -  `Browser Type: *` Chrome
 -  `Browser Version: *` 59
 -  `OS: *` Chrome OS

**Comment by EamonKavanagh** [comment]
+1 to this issue- I'm not sure a user using ChromeVox will be able to use md-autocompletes as they won't know which option they've highlighted.

We've also had reports of this with JAWS.

**Comment by Splaktar** [comment]
This is not reproducible with VoiceOver for macOS.

**Comment by Splaktar** [comment]
I can reproduce this on ChromeOS Version 68.0.3440.87 (Official Build) beta (64-bit) with the latest AngularJS Material.

It appears that Chromevox does not support the following:
```html
<div class="md-visually-hidden" role="status" aria-atomic="true" aria-live="assertive">Florida</div>
```
It should read "Florida" but instead reads "Status".

**Comment by Splaktar** [comment]
It's not clear that Chromevox supports the `status` role.
[MDN's docs on status compatibility](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_status_role#Compatibility) are TODO.

The MDN site also has a link to [ARIA Best Practices – Implementing Live Regions](http://www.w3.org/TR/wai-aria-practices/#LiveRegions) which has been removed from the latest WAI-ARIA docs 😞 
...
OK, I fixed MDN's link. 

Docs for [role="status"](https://www.w3.org/TR/wai-aria-1.1/#status) and docs for [aria-live and aria-atomic can be found here](https://www.w3.org/TR/wai-aria-1.1/#attrs_liveregions). These point to the 1.1 Published Recommendation.

**Comment by Splaktar** [comment]
I submitted a request for documentation of what Live Region roles/attributes are supported in Chromevox here:
https://groups.google.com/forum/#!topic/chromevox-discuss/M2gb3CFsois

I also opened up a Chromium bug here:
https://bugs.chromium.org/p/chromium/issues/detail?id=873466

**Comment by hintzed** [comment]
I did reproduce the issue with Chrome 69 and was able to verify that it seems fixed in Chrome 70 build. It will now read the message, followed by `Status` on Chrome 70 with ChromeVox on a Chromebook.

**Comment by Splaktar** [comment]
@hintzed thank you very much for testing this out! That's good news. We'll have some additional autocomplete fixes in 1.1.11 as well.
